automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodImportSpec.scala

```diff
@@ -14,15 +14,16 @@ class MethodImportSpec extends FreeSpec with Matchers with WebBrowserSpec with W
       val user = UserPool.chooseProjectOwner
       implicit val authToken: AuthToken = user.makeAuthToken()
       withCleanBillingProject(user) { billingProject =>
-        withWorkspace(billingProject, "Copy_method_config_from_workspace_src") { sourceWorkspaceName =>
-          withWorkspace(billingProject, "Copy_method_config_from_workspace_dest") { destWorkspaceName =>
+        withWorkspace(billingProject, "MethodImportSpec_workspace_src") { sourceWorkspaceName =>
+          withWorkspace(billingProject, "MethodImportSpec_workspace_dest") { destWorkspaceName =>
             withMethod("MethodImportSpec_import_from_workspace", MethodData.SimpleMethod, 1) { methodName =>
               val method = MethodData.SimpleMethod.copy(methodName = methodName)
               api.methodConfigurations.createMethodConfigInWorkspace(billingProject, sourceWorkspaceName,
                 method, method.methodNamespace, method.methodName, 1, Map.empty, Map.empty, method.rootEntityType)
 
               withWebDriver { implicit driver =>
                 withSignIn(user) { listPage =>
+                  api.workspaces.waitForBucketReadAccess(billingProject, destWorkspaceName)
```

Comment:
fix for this issue?? https://fc-jenkins.dsp-techops.broadinstitute.org/view/Testing/view/Test%20Runners/job/firecloud-ui-real-test-runner/107/artifact/failure_screenshots/2018-05-11_12-40-12-877_MethodImportSpec.png